internal/gengapic/gengrpc.go

```diff
@@ -32,7 +32,6 @@ func (g *generator) genGRPCMethods(serv *descriptor.ServiceDescriptorProto, serv
 
 	methods := append(serv.GetMethod(), g.getMixinMethods()...)
 	for _, m := range methods {
-		g.methodDoc(m)
```

Comment:
Should this also be removed from `genGRPCMethod `?

Reply:
We can keep it for sure, but the `gRPCClient` is internal and would be a duplicate of what is on the wrapper. WDYT? Happy to go either way.

Reply:
I would keep the method comments on both the wrapper and the internals just to help whoever peruses the code (mostly us?). Since they're autogen'd, they'll always stay in sync.

Not a biggie either way.

Reply:
+1 for removing it from gRPC client as well.

Reply:
@codyoss what say ye? I'm leaning towards removing it (as I've done).

Reply:
Cody +1 my previous comment, so we will omit generated docs on the internal methods for now. Resolving.